.github/workflows/crowdin-pull.yml

```diff
@@ -74,6 +75,8 @@ jobs:
       # validation failures in the PR
       - name: Normalize and validate
         run: npm run translate
+      - name: Prepare metadata directories
```

Comment:
In addition to my more general comment about where this functionality should live, this action step should be named a bit more specifically, like "Prepare Fastlane metadata directories" or "Prepare app store metadata" or something.

Reply:
Have renamed